src/styles/main.scss

```diff
@@ -0,0 +1,60 @@
+$height_watch: 80vmin;
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  margin: auto;
+}
+
+.watch {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: $height_watch;
+  height: $height_watch;
+  border-radius: 50%;
+  border: 9px dotted #000;
+
+  &__second-hand {
+    position: absolute;
+    background-color: green;
+    width: 9px;
+    height: calc(#{$height_watch} / 2);
+    top: 50%;
+    left: 50%;
+    transform-origin: bottom center;
+    animation: rotate 5s linear infinite;
```

Comment:
It would be great to create variables for animation duration and colors.
![image](https://github.com/mate-academy/layout_stop-watch/assets/96409563/97d8faa2-a61d-496e-b07b-ff049814c0e0)

Reply:
done